sdk/ml/azure-ai-ml/azure/ai/ml/_schema/assets/asset.py

```diff
@@ -21,6 +22,7 @@ class AssetSchema(ResourceSchema):
     version = VersionField()
     creation_context = NestedField(CreationContextSchema, dump_only=True)
     latest_version = fields.Str(dump_only=True)
+    auto_delete_setting = ExperimentalField(NestedField(AutoDeleteSettingSchema))
```

Comment:
Does this applly to all asset types?  Or it's just data asset?

Reply:
Auto delete setting should be a base property for all assets, but only be used in data asset at first.